tests/interfaces_test.py

```diff
@@ -40,6 +40,7 @@
 Ethernet116  etp30
 Ethernet120  etp31
 Ethernet124  etp32
+Ethernet128  etp33
```

Comment:
why add this new entry?

Reply:
internal interface for single asic, none were present initially

Reply:
there is no interface interfaces in single ASIC

Reply:
will remove